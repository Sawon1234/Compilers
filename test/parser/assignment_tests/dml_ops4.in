x := true + false;
y := 3 | 5;
z := "hello" < "goodbye";
