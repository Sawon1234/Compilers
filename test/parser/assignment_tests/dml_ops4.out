x := true . plus < > ( false ) ; y := 3 . or < > ( 5 ) ; z := "hello" . lessThan < > ( "goodbye" , true ) ;